src/lib/activity-store.js

```diff
@@ -4,10 +4,14 @@ import Elasticsearch from '@elastic/elasticsearch';
 class ActivityStore {
 
   constructor(esIndex){
-    this.user = "test";
-    this.pass = "test";
+    this.user = process.env.ELASTICSEARCH_USERNAME || "";
+    this.pass = process.env.ELASTICSEARCH_PASSWORD || "";
```

Comment:
the value doesn't need a default, the property not existing will evaluate to "undefined" which is falsey

Reply:
ta